### ISSUE TITLE
Can't deploy signed .ipa to the app store because of the missing key in plist

### ISSUE THREAD

**Issue by Merraclius** [AgoraIO-Extensions/Agora-Flutter-SDK#778]
**Describe the bug**
Faced with the problem when I can't deploy to the App Store application.
When using version 6.0.0. during deployment to the App Store, I got the following errors:
```
*** Error: Asset validation failed The bundle 'Payload/Runner.app/Frameworks/iris_event_handler.framework' is missing plist key. The Info.plist file is missing the required key: CFBundleShortVersionString. Please find more information about CFBundleShortVersionString at https://developer.apple.com/documentation/bundleresources/information_property_list/cfbundleshortversionstring (ID: ***) (90057)
 {
    NSLocalizedDescription = "Asset validation failed";
    NSLocalizedFailureReason = "The bundle 'Payload/Runner.app/Frameworks/iris_event_handler.framework' is missing plist key. The Info.plist file is missing the required key: CFBundleShortVersionString. Please find more information about CFBundleShortVersionString at https://developer.apple.com/documentation/bundleresources/information_property_list/cfbundleshortversionstring (ID: ***)";
    "original_server_error" =     {
        code = "STATE_ERROR.VALIDATION_ERROR.90057";
        detail = "The bundle 'Payload/Runner.app/Frameworks/iris_event_handler.framework' is missing plist key. The Info.plist file is missing the required key: CFBundleShortVersionString. Please find more information about CFBundleShortVersionString at https://developer.apple.com/documentation/bundleresources/information_property_list/cfbundleshortversionstring";
        id = "***";
        status = 409;
        title = "Asset validation failed";
    };
}
*** Error: Asset validation failed This bundle Payload/Runner.app/Frameworks/iris_event_handler.framework is invalid. The Info.plist file is missing the required key: CFBundleVersion. Please find more information about CFBundleVersion at https://developer.apple.com/documentation/bundleresources/information_property_list/cfbundleversion (ID: ***) (90056)
 {
    NSLocalizedDescription = "Asset validation failed";
    NSLocalizedFailureReason = "This bundle Payload/Runner.app/Frameworks/iris_event_handler.framework is invalid. The Info.plist file is missing the required key: CFBundleVersion. Please find more information about CFBundleVersion at https://developer.apple.com/documentation/bundleresources/information_property_list/cfbundleversion (ID: ***)";
    "original_server_error" =     {
        code = "STATE_ERROR.VALIDATION_ERROR.90056";
        detail = "This bundle Payload/Runner.app/Frameworks/iris_event_handler.framework is invalid. The Info.plist file is missing the required key: CFBundleVersion. Please find more information about CFBundleVersion at https://developer.apple.com/documentation/bundleresources/information_property_list/cfbundleversion";
        id = "***";
        status = 409;
        title = "Asset validation failed";
    };
}
```

**To Reproduce**
1. Build the signed iOS application
2. Execute `xcrun "altool" "--validate-app" "-f" "./Runner.ipa" "--type" "ios"`

**Expected behavior**
The application should be valid and can be deployed to the App Store

**Additional context**
Application built in the Xcode 14.
Found a similar problem, for another framework - https://github.com/AgoraIO-Extensions/Agora-Flutter-SDK/issues/609

**Comment by littleGnAl** [comment]
Thanks for reporting, we will fix it in the next release.
You can temporarily edit the `YOUR/APP/ios/Pods/AgoraIrisEventFlutter_iOS/iris_event_handler.xcframework/ios-arm64_armv7/iris_event_handler.framework/Info.plist` to add the `CFBundleShortVersionString `/`CFBundleVersion ` in the `Info.plist` file. e.g.,
```xml
<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
<plist version="1.0">
<dict>
...
	<key>CFBundleShortVersionString</key>
	<string>1.5.0</string>
	<key>CFBundleVersion</key>
	<string>1.5.0</string>
...
</dict>
</plist>
```

**Comment by federico-amura-kenility** [comment]
i have the same issue, but i cant do that temp fix becouse i build from Bitrise/codemagic solution. Any solution?

**Comment by littleGnAl** [comment]
@federico-amura-kenility This fix will land on the next release.

**Comment by littleGnAl** [comment]
This should be fixed on `agora_rtc_engine: ^6.1.0`
